test/ray_julia_jll/runtests.jl

```diff
@@ -16,10 +16,10 @@ end
     include("address.jl")
     include("objectid.jl")
 
-    setup_ray_head_node() do
+    setup_ray_head_node_basic() do
         # GCS client only needs head node
         include("gcs_client.jl")
-        setup_core_worker() do
+        setup_core_worker_basid() do
```

Comment:
```suggestion
        setup_core_worker_basic() do
```

Reply:
Thanks. Of course the last minute change has a typo